src/components/DroughtThresholds.vue

```diff
@@ -1861,6 +1890,8 @@ $usgsBlue: #032a56;
 
 .visible{
   visibility: visible;
+  opacity: 1;
```

Comment:
I *think* I accomplished this just by setting the min-height of the text container so that everything lines up, but unsure how consistent that will work on other browsers/sizes? Also, didn't work for the mobile layout?

Reply:
If I change the `.textbox` `min-width` setting to `min-height` instead then it works for me on mobile. Only it depends on screen size -- I have to bump it up to `min-height: 160px` to catch those longer blocks of text too. I think it could be fine to leave as is on mobile b/c having the text vertically centered between the title and chart looks nice.

Reply:
Does leave it as is mean keep it as my newer `min-height: 100px;` or take that line out?

Reply:
I meant take that line out, but I think you should do what you prefer.